shell/edit/persistentvolume/index.vue

```diff
@@ -39,18 +40,23 @@ export default {
     UnitInput
   },
 
-  mixins: [CreateEditView],
+  mixins: [CreateEditView, ResourceManager],
 
-  async fetch() {
-    const storageClasses = await this.$store.dispatch('cluster/findAll', { type: STORAGE_CLASS });
-
-    await this.$store.dispatch('cluster/findAll', { type: PVC });
+  fetch() {
+    if (this.mode === _EDIT) {
+      this.secondaryResourceData.data[PVC] = {
```

Comment:
PVCs are namespaced, so this needs a `namespace` property somewhere (currently the request sent out ends in `/null` - see comment in mixin)